projects/core/src/cms/store/reducers/index.ts

```diff
@@ -45,6 +47,7 @@ export function getReducers(): ActionReducerMap<CmsState> {
       }),
     }),
     component: entityLoaderReducer(COMPONENT_ENTITY),
+    components: entityReducer(COMPONENT_ENTITY, fromComponentsReducer.reducer),
```

Comment:
For readablity, it would be nice to add here a comment that `component` state is deprecated and `components` is the recommended to stay